string zone
string relationship
string[] node_objects
geometry_msgs/Pose pose
shape_msgs/SolidPrimitive primitive